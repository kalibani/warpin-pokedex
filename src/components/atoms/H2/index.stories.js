// Stories H2 Component
// --------------------------------------------------------
import React, { Fragment } from 'react';
import { storiesOf } from '@storybook/react';
import H2 from './index';
import Notes from './readme.md';

storiesOf('Atoms', module).add('H2', () => (
  <H2 />
), {
  info: { inline: true, header: false, text: Notes }
});
