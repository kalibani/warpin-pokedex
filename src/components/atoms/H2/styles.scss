/* ==========================================================================
Styles for H2 component
========================================================================== */

// import mixin and variables
@import "../../../assets/styles/variables";

// Start styles H2 component
.a-h2 {
  font-size: 34px;
  font-weight: 600;
  color: $white;
  margin: 0;
}
