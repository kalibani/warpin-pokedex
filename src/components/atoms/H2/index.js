// H2 Component
// --------------------------------------------------------

import React from 'react';
import PropTypes from 'prop-types';
import './styles.scss';

const H2 = ({ propsName }) => (
  <tag>
    Functional Component (Stateless Component)
  </tag>
);

H2.propTypes = {
  propsName: PropTypes.string
};

H2.defaultProps = {
  propsName: ''
};

export default H2;
