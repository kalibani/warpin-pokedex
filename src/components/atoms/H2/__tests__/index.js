// Test H2 Component
// --------------------------------------------------------

import React from 'react';
import H2 from '../../index';
import renderer from 'react-test-renderer';

test('H2 is match the snapshot', () => {
const component = renderer.create(
H2,
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});
