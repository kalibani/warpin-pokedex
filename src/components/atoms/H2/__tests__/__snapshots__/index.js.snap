// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 is match the snapshot 1`] = `null`;
